README.md

```diff
@@ -110,11 +112,25 @@ If you followed the instructions for `CKAN load testing` and `Harvester testing`
 3. when there are database DDL changes, use following steps to generate migration scripts and update database:
 
     ```bash
-    docker compose db up
+    docker compose up -d db
+    docker compose run app flask db stamp head
```

Comment:
Can someone confirm this is correct? I ran into the error below, but this allowed me to proceed.

`ERROR [flask_migrate] Error: Target database is not up to date.`

Reply:
we need to run 'flask db upgrade' before 'flask db stamp head' to make sure changes be applied to database. And the 'flask db stamp head' command is optional I think.

Reply:
Thanks, Jin! Let me try that out on a new branch to confirm.

Reply:
I can't replicate it. Maybe I don't need to add the migration and it was just something on my local? Will bring this question to sync.